src/runtime/particle-execution-context.ts

```diff
@@ -252,7 +252,7 @@ export class ParticleExecutionContext {
     // Create new handles and disable the handles of the old particles
     oldParticle.handles.forEach((oldHandle) => {
       this.createHandle(particle, oldParticle.spec, id, oldHandle.name, oldHandle.storage, handleMap, registerList);
-      if (oldHandle instanceof HandleOld) oldHandle.disable(oldParticle);
```

Comment:
this change does not seem necessary, since they have the same method.

Reply:
This was my workaround when HandleNG didn't have the disable function (only call disable when it's HandleOld). Now since both of them have the method, I changed it back to just calling the method without the if check.